cli/jobs/single-step/java/iris/src/test/java/com/microsoft/aml/auth/AMLAuthenticationTest.java

```diff
@@ -0,0 +1,25 @@
+package com.microsoft.aml.auth;
+
+import org.junit.Test;
+import org.nd4j.linalg.io.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is a test class for Authentication
+ * @author Mufy, Abe
+ * @Date 7/1/2022
+ */
+public class AMLAuthenticationTest {
+
+	private static final Logger log = LoggerFactory.getLogger(AMLAuthenticationTest.class);
+
+	@Test
+	public void testAMLAuthentication() throws Exception {
```

Comment:
We should have at least one positive and one negative test

Reply:
Added few negative tests.